client/layout/error.jsx

```diff
@@ -48,5 +50,12 @@ export function retry( chunkName ) {
 export function show( context, chunkName ) {
 	log( 'Chunk %s could not be loaded', chunkName );
 	analytics.mc.bumpStat( 'calypso_chunk_error', chunkName );
+	context.store.dispatch( {
```

Comment:
We also dispatch this action here: https://github.com/Automattic/wp-calypso/blob/master/client/sections-middleware.js#L68 - does it make sense to combine both calls?